src/index.ts

```diff
@@ -17,17 +15,14 @@ const renderUsers = async (config: ApiConfig) => {
   });
 };
 
-const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
-
-  renderUsers(config.api);
+const app = async () => {
+  await renderUsers();
```

Comment:
There is no point in using async/await, because there is nothing to execute after 'renderUsers' function.
But, the task is not about it, so approved.